webpack-demo/src/listFunctions.js

```diff
@@ -0,0 +1,93 @@
+// eslint-disable-next-line import/no-cycle
+import { showList } from './index.js';
+
+// eslint-disable-next-line import/no-mutable-exports
+export let listData = [{}];
```

Comment:
- [ ] Please make sure that when the app is first opened it doesn't display the following task which just says "undefined". 
![image](https://user-images.githubusercontent.com/62290545/213525149-4c05b682-aebe-494c-90e5-e2585f457ca5.png)